src/Header.tsx

```diff
@@ -60,6 +60,7 @@ export default function Header() {
         fontSize: '2.5em',
         fontWeight: 400,
         textAlign: 'left',
+        display: 'none',
```

Comment:
A temporary measure I assume!

What's the plan @dmfalke for the site header? I guess it's not a `web-eda` issue.

Reply:
Oh yes, this code is not in decent shape right now lol. There's a prop now for hiding site headers that I can use.

Reply:
Updated in: https://github.com/VEuPathDB/web-eda/pull/1613/commits/a1ab96716bf0e6393dd729bfcce40e3ba9a144cf